Frontend/src/data/cluster-data.tsx

```diff
@@ -0,0 +1,49 @@
+import { IClusterDetails } from "@/types/cluster-types";
+
+const sampleNodeData = [
```

Comment:
Why specifically these IP addresses?

Reply:
these are sample datas, i removed this file, we're not using these anymore

Reply:
Ok